docs/wxm-devices/D1-bundle/D1-connect-and-claim.mdx

```diff
@@ -0,0 +1,202 @@
+
+  </div>
+  <span>Download our WeatherXM apps from their respective app store. Once you have your WeatherXM app, login to your account (you have to create a new WeatherXM account).</span>
+</div>
+
+<hr/>
+
+
+### WeatherXM App - Claim Device
+### Tap the "+" button
+
+<div class="explanation">
+  <img src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/pus-button.svg")} />
+  <div>
+    <h4>Start the Claiming Process</h4>
+    <p>
+      Once you've logged in to your WeatherXM app, you will see an empty device list (assuming this is your first station setup). At the bottom right corner, you will see a "+" button. Tap it to start the claiming process of your station. Once you've done this, you'll be able to view all data of this station.
+    </p>
+  </div>
+</div>
+
+:::caution
+Your first rewards will appear after 24 hours of the station being online.
+:::  
+
+<div class="explanation">
+        <img style={{width:"calc(100%*1/3 - 10px)"}} src={useBaseUrl("/img/wxm-devices/D1-bundle/select-your-station.png")} />
+  <div style={{width:"calc(100%*2/3 - 10px)"}} >
+
+
+#### Follow the steps from the app to successfully claim your D1 Gateway.
+
+
+
+
+  </div>
+</div>
+
+<hr/>
+
+
+<div style={{ display: 'flex', alignItems: 'center' }}>
+  <img 
+    src={useBaseUrl("/img/wxm-devices/D1-bundle/D1-Location.png")} 
+    style={{ width: '250px', height: 'auto', objectFit: 'contain', marginRight: '20px' }} 
+  />
+  <div>
+    <h3>Confirm station installation Location & Claim your Device</h3>
+    <h4>Select the Location that you'll Install your Station</h4>
+    <p>Allow access to your location so that the WeatherXM app knows where you are planning to install your weather station. Doing so will make it easier to complete this step.</p>
+    <p>Alternatively, you can find the location manually using the map.</p>
+    <p>Once you've found the right location, tap on the checkbox to confirm and then tap on the "Claim" button.</p>
+    <p>All you need to do now, is to add your compatible wallet so you can receive WXM rewards. To do so, please follow the instructions in the chapter <a href="/wallet/add-edit-wallet-address#how-to-create-wallet-on-metamask">"Add/edit wallet address"</a>.</p>
+  </div>
+</div>
+
+   
+   
+**That's it! You've now claimed your new weather station and you'll be able to see its data on your WeatherXM app!**
+
+
+:::note
+We do not track you, nor do we collect any data about your location.
+:::
+
+:::caution
+Make sure you define the location that you will install your station as precisely as possible, as this is important for the quality of data you will share with the network and thus your overall WXM rewards.
+:::
+
+:::caution
+Your first rewards will appear after 24 hours of the station being online.
+:::  
+
+
+
+:::info 
+Usually, it takes up to 30 minutes for the station to appear on the [explorer](https://explorer.weatherxm.com/stats)
+:::
+
+
+
+
```

Comment:
Remove these lines. Just copy the same code from M5 as seen here: https://docs.weatherxm.com/wxm-devices/wifi-m5-bundle/m5-connect-and-claim#confirm-station-installation-location--claim-your-device and just replace the image with the updated one.